roles/heat/templates/etc/heat/heat.conf

```diff
@@ -55,6 +55,7 @@ num_engine_workers = {{ heat.engine_workers }}
 
 [database]
 connection=mysql://heat:{{ secrets.db_password }}@{{ endpoints.db }}/heat?charset=utf8
+max_overflow=1000
```

Comment:
Would it be better to make this a var in defaults/main.yml opposed to setting to 1000 in the template?